.github/workflows/ci.yml

```diff
@@ -2,6 +2,29 @@ name: "CI"
 
 on:
   workflow_dispatch:
+    inputs:
+      enableUpterm:
+        description: "Enable the Upterm server for debugging workflows in-flight"
+        required: true
+        default: false
+        type: boolean
+      breakOnJob:
+        description: "Break on a specific job, to be used with the `enableUpterm` input"
+        required: false
+        type: choice
+        options:
+          - "changes"
+          - "pkgdb-dev"
+          - "cli-dev"
+          - "nix-build"
+          - "trigger-flox-installers-workflow"
+          - "report-failure"
+          - "nix-build-bats-tests"
+      allowedUptermUsers:
+        description: "Comma-separated list of GitHub users allowed to access the Upterm server"
```

Comment:
Looks like we narrowed the list from a previous commit but I had to read https://github.com/owenthereal/action-upterm/tree/main?tab=readme-ov-file#use-registered-public-ssh-keys to see why and what `limit-access-to-actor` does:
```suggestion
        description: "Comma-separated list of GitHub users allowed to access the Upterm server in addition to the current actor"
```

Although it's a bit odd that this surfaces in the UI each time:

![image](https://github.com/user-attachments/assets/bab6d1fb-94af-4fd8-86c4-619b188dcbb5)